schema/tracer_provider.json

```diff
@@ -123,6 +126,15 @@
                 }
             }
         },
+        "ExperimentalProbabilitySampler": {
+            "type": ["object", "null"],
+            "additionalProperties": false,
+            "properties": {
+                "ratio": {
+                    "type": ["number", "null"]
```

Comment:
Could add `"minimum": 0`, `"maximum": 1`: https://json-schema.org/understanding-json-schema/reference/numeric#range

Or maybe `"exclusiveMinimum": 0` since the spec says the minimum valid ratio is `2^-56`: https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/trace/sdk.md#probabilitysampler-sampler-configuration